cypress.config.js

```diff
@@ -2,6 +2,9 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    viewportWidth: 1500,
```

Comment:
The properties 'viewportWidth', 'viewportHeight' and 'baseUrl' are not valid properties of the 'e2e' configuration object. They should be placed directly inside the main configuration object.